packages/headless/src/controllers/breadcrumb-manager/headless-breadcrumb-manager.ts

```diff
@@ -1,4 +1,4 @@
-import {Engine} from '../../app/engine';
+import {Engine} from '../../app/headless-engine';
```

Comment:
Readjusting these imports from the last PR. Initially, I wanted `Engine` to be the name of the core interface, with the concrete instance being called e.g. `SearchEngine`.

However, this would be breaking because it means the publically exported `Engine` interface would no longer be associated with the Search API. Instead `Engine` is back to any engine using the Search API client, and `CoreEngine` is the name of the core engine interface.